finetune/nn/auxiliary.py

```diff
@@ -121,7 +125,22 @@ def dense_with_custom_init(input_tensor,
             return z
 
     else:
-        return tf.layers.dense(input_tensor, output_dim, activation=activation, name=name, kernel_initializer=kernel_initializer)
+        if transpose_b:
```

Comment:
Could we always use the path that doesn't use tf.layers.dense?  And only conditionally transpose?

Reply:
Yeah I could do that. Just need to do a bit of testing to make sure the naming is correct